101-vm-ws2016-visualstudio2019/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "itemDisplayName": "Deploy a Windows SV VM with Visual Studio.",
```

Comment:
SV?

Reply:
It was meant to be an abbreviation of Server, but now looking, doesn't looks good. Changed!